modules/persistent-storage-local-install.adoc

```diff
@@ -49,3 +49,85 @@ $ oc annotate project local-storage openshift.io/node-selector=''
 .. Click *Subscribe*.
 
 . Once finished, the Local Storage Operator will be listed in the *Installed Operators* section of the web console.
+
+.. Navigate to *Operators* -> *OperatorHub*.
+
+.. Type *Local Storage* into the filter box to locate the Local Storage Operator.
+
+.. Click *Install*.
+
+.. On the *Create Operator Subscription* page, select *A specific namespace on the cluster*. Select *local-storage* from the drop-down menu.
+
+.. Adjust the values for the *Update Channel* and *Approval Strategy* to the desired values.
+
+.. Click *Subscribe*.
+
+. Once finished, the Local Storage Operator will be listed in the *Installed Operators* section of the web console.
+
+.From the CLI
+. Install the Local Storage Operator from the CLI.
+
+.. Create an object YAML file to define a Namespace, OperatorGroup, and Subscription for the Local Storage Operator,
+such as `local-storage.yaml`:
++
+.Example local-storage
+[source,yaml]
+----
+  apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: local-storage
+  ---
+  apiVersion: operators.coreos.com/v1alpha2
+  kind: OperatorGroup
+  metadata:
+    name: local-operator-group
+    namespace: local-storage
+  spec:
+    targetNamespaces:
+      - local-storage
+  ---
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: local-storage-operator
+    namespace: local-storage
+  spec:
+    channel: "4.2"
+    installPlanApproval: Automatic
+    name: local-storage-operator <1>
+    source: redhat-operators <2>
+    sourceNamespace: openshift-marketplace <3>
+----
+<1> Name of the Operator to subscribe to.
+<2> Name of the CatalogSource that provides the Operator.
+<3> Namespace of the CatalogSource. Use `openshift-marketplace` for the default
+OperatorHub CatalogSources.
```

Comment:
Essentially the only field that user should edit here is to change the channel to match OCP version. Everything else user *must* not touch.

Reply:
In here, the end users should care about the spec.channel and spec.installPlanApproval same with ui installation.

Reply:
Fair point, @gnufied. I was pulling the annotation descriptions from our Operator docs ([link](https://bz1800649--ocpdocs.netlify.com/openshift-enterprise/latest/operators/olm-adding-operators-to-cluster.html#olm-installing-operator-from-operatorhub-using-cli_olm-adding-operators-to-a-cluster)).
But we don't want to give any impression that those fields are customizable, so I removed them. I also added a description for channel (to match OCP version).

@qinpingli I didn't add a note to spec.installPlanApproval as I wasn't sure we wanted to have them use anything but `Automatic`. Please suggest wording for this field if you feel strongly that a description is needed.